localstack-core/localstack/services/events/rule.py

```diff
@@ -38,7 +52,7 @@ def __init__(
         event_pattern: Optional[EventPattern] = None,
```

Comment:
nit: `EventPattern | None` is now the natively supported syntax since Python 3.10 https://peps.python.org/pep-0604/